packages/runtime/src/widgets/DataGrid/DataGrid.tsx

```diff
@@ -168,14 +171,16 @@ export const DataGrid: React.FC<GridProps> = (props) => {
               }
             : undefined
         }
+        scroll={
+          values?.scrollable ? { y: values.scrollHeight || 150 } : undefined
+        }
```

Comment:
Having two properties seems redundant here. Let's just expose the scroll config object